File: app/src/main/java/wxm/dofcalculator/ui/help/ACHelp.java
package wxm.dofcalculator.ui.help;


import android.os.Bundle;

import wxm.androidutil.Switcher.ACSwitcherActivity;

/**
 * help UI
 */
public class ACHelp extends ACSwitcherActivity<FrgHelp> {
    public static final String STR_HELP_TYPE            = "HELP_TYPE";
    public static final String STR_HELP_START           = "help_start";

    @Override
    protected void initUi(Bundle savedInstanceState)    {
        super.initUi(savedInstanceState);
        addFragment(new FrgHelp());
    }
}

File: app/src/main/java/wxm/dofcalculator/ui/help/ACHelp.kt
package wxm.dofcalculator.ui.help


import android.os.Bundle

import wxm.androidutil.Switcher.ACSwitcherActivity

/**
 * help UI
 */
class ACHelp : ACSwitcherActivity<FrgHelp>() {
    override fun setupFragment(bundle: Bundle?) {
        addFragment(FrgHelp())
    }

    companion object {
        const val STR_HELP_TYPE = "HELP_TYPE"
        const val STR_HELP_START = "help_start"
    }
}
